test/integration/hds_integration_test.cc

```diff
@@ -118,7 +118,7 @@ class HdsIntegrationTest : public HttpIntegrationTest,
   // one HTTP health_check
   envoy::service::discovery::v2::HealthCheckSpecifier makeHttpHealthCheckSpecifier() {
     envoy::service::discovery::v2::HealthCheckSpecifier server_health_check_specifier_;
-    server_health_check_specifier_.mutable_interval()->set_seconds(1);
+    server_health_check_specifier_.mutable_interval()->set_nanos(100000000);
```

Comment:
Why not still use seconds here for readability? Same elsewhere? Or perhaps a constant such as MaxTimeout to make it clear that we don't care about the timeout?

Reply:
ah ok. Alright, some type of constant or comment might still be nice here since it's hard to read how much time this is.